package/kedro_viz/launchers/cli/utils.py

```diff
@@ -104,16 +104,6 @@ def create_shareableviz_process(
         viz_deploy_process.terminate()
 
 
-def display_cli_message(msg, msg_color=None):
```

Comment:
Why was this moved?

Reply:
Earlier this was used only in cli. In this PR I tried using it in dev mode in [`server.py`](https://github.com/kedro-org/kedro-viz/pull/2113/files#diff-096675efe9d411d7fb88ddc3a449afef347ba47fb5a6aa5812f8602738e240efR179) 
To avoid circular dependency, this needs to be moved